doc/howtos/booting-and-installation.rst

```diff
@@ -127,6 +127,58 @@ includes a checksum verification, you can also use the traditional :command:`dd`
 Unplug the removable media from your development system and you're ready to plug
 it into your target system.
 
+Installing to internal media on generic x86 EFI platforms
+=========================================================
+
+This section applies to those cases where the device is x86 compatible (ex: Gigabyte NUC).
+The simpler way is to follow these steps:
+
+#. Create a Bootable Media using an both an image that is compatible with the device
+   and a supported removable media (SD card or USB stick).
+#. Boot the device with the removable media created at the previous step.
+#. Once the device is booted, identify the booted media, for example by executing::
+
+      $ mount
+
+   and locating in its output which partition is mounted ``on /``.
+   This will be a partition belonging to the source block device.
+   Example::
+
+      /dev/sdb3 on / type ext4 (rw,realtime,i_version,data=ordered)
+
+   Shows that the root partition is on `/dev/sdb3`, therefore the source block device will
+   be `/dev/sdb`.
+#. Identify the block device representing the internal storage.
+   This part is really specific to each platfrom: typically the internal storage is
+   associated to `/dev/sda` for hard disks and SSD units and to `/dev/mmcblk0` for eMMC
+   devices, but it should be confirmed against the board manual.
```

Comment:
I don't know. Maybe lspci & lsusb would help. But not blkid, afaik it relies on logical information present on the block devices. But they might be empty/corrupted. In reality I expect one knows what local storage is present on a certain device.